cli/command/image/list.go

```diff
@@ -59,6 +61,9 @@ func NewImagesCommand(dockerCLI command.Cli) *cobra.Command {
 	flags.StringVar(&options.format, "format", "", flagsHelper.FormatHelp)
 	flags.VarP(&options.filter, "filter", "f", "Filter output based on conditions provided")
 
+	flags.BoolVar(&options.tree, "tree", false, "List multi-platform images tree [experimental, behavior may change]")
```

Comment:
There's an annotation for experimental CLI commands and flags. I believe it hides it from `--help` in that case though.

For buildx experimental flags we add a parenthesized all-caps `(EXPERIMENTAL)` suffix.
https://github.com/docker/buildx/blob/e2f6808457ab9d8de4a85f0f26b0028411ef20fc/util/cobrautil/cobrautil.go#L22-L31

```suggestion
	flags.BoolVar(&options.tree, "tree", false, "List multi-platform images tree (EXPERIMENTAL)")
	flags.SetAnnotation("tree", "experimentalCLI", nil)
```

Reply:
This annotation is for features that require `--experimental` flag enabled on the daemon side though.

I can adjust the suffix though

Reply:
Sorry for the back and forth... I did some digging and it turns out that we strip the `(EXPERIMENTAL)` suffix from the generated buildx doc strings. https://github.com/docker/buildx/blob/e2f6808457ab9d8de4a85f0f26b0028411ef20fc/docs/generate.go#L30-L50

And we use the `experimentalCLI` annotation to [generate an experimental badge](https://github.com/docker/docs/blob/main/layouts/_default/cli.html#L140-L142) on the documentation site (via [docker/cli-docs-tool](https://github.com/docker/cli-docs-tool/blob/main/clidocstool_yaml.go#L178-L180)).

<img width="664" alt="image" src="https://github.com/user-attachments/assets/cea1703f-3220-40a8-8ef8-c1a9f6c9a6e4">

so tl;dr: keep the annotation, don't put "experimental" in the usage desc.

Reply:
Added the annotation

Reply:
I think we want to remove the experimental callout in the usage description too, otherwise it would appear double in the docs site.

```suggestion
	flags.BoolVar(&options.tree, "tree", false, "List multi-platform images tree")
```

Reply:
Hmm, but that won't make it visible in the CLI itself

Reply:
yeah, hence the buildx hack. I'm fine to duplicate it.